core/tests/webdriverio_utils/ExplorationEditorPage.js

```diff
@@ -44,20 +44,32 @@ var ExplorationEditorPage = function() {
   */
   var commitMessageInput = $('.e2e-test-commit-message-input');
   var confirmPublish = $('.e2e-test-confirm-publish');
-  var expCategoryDropdownElement = $('.e2e-test-exploration-category-dropdown');
+  var expCategoryDropdownElement = $(
+    '.e2e-test-exploration-category-metadata-modal');
   var expLanguageSelectorElement = $('.e2e-test-exploration-language-select');
+  var expLanguageSelectorElementModal = $(
+    '.e2e-test-exploration-language-select-modal');
   var explorationSaveModalElement = $('.e2e-test-exploration-save-modal');
   var expTagsSelectionChoiceElementsSelector = function() {
     return $$('.select2-selection__choice');
   };
   var expObjective = $('.e2e-test-exploration-objective-input');
   var expTags = $('.e2e-test-tags');
   var expTitle = $('.e2e-test-exploration-title-input');
+  var expTitleInPostPublishModal = $(
+    '.e2e-test-exploration-title-input-modal');
+  var expObjectiveInPostPublishModal = $(
+    '.e2e-test-exploration-objective-input-modal');
   var explorationMetadataModalHeaderElement = $(
     '.e2e-test-metadata-modal-header');
   var modalContentElement = $('.modal-content');
   var promptModalElement = $('.e2e-test-save-prompt-modal');
-  var selectionRenderedElement = $('.select2-selection__rendered');
+  var languageChoiceOptionElement = function(language) {
+    return $$(
+      `.e2e-test-exploration-language-select-lan=${language}`).first();
```

Comment:
General way of using a`$$` selector is as follows:
```
var languageChoiceOptionElement = function(language) {
    return $$(
      `.e2e-test-exploration-language-select-lan=${language}`);
};

var  languageChoiceOption = await languageChoiceOptionElement(language)[0];
```

Please update it, thanks.

Reply:
done